crypto/arm64cpuid.pl

```diff
@@ -151,7 +151,63 @@
 	lsr	w0,w0,#31
 	ret
 .size	CRYPTO_memcmp,.-CRYPTO_memcmp
+
+.globl	_armv8_rng_probe
+.type	_armv8_rng_probe,%function
+_armv8_rng_probe:
+	mrs	x0, s3_3_c2_c4_0	// rndr
+	mrs	x0, s3_3_c2_c4_1	// rndrrs
+	ret
+.size	_armv8_rng_probe,.-_armv8_rng_probe
+___
+
+sub gen_random {
+my $rdop = shift;
+my $rand_reg = $rdop eq "rndr" ? "s3_3_c2_c4_0" : "s3_3_c2_c4_1";
+
+print<<___;
+.globl	OPENSSL_${rdop}_asm
+.type	OPENSSL_${rdop}_asm,%function
+.align	4
+OPENSSL_${rdop}_asm:
+	mov	x2,xzr
```

Comment:
It is not too hard to read this - but it wouldnt hurt to add a few comments related to the parameters

Reply:
comments added. resolving